Geometry/MTDCommonData/test/DD4hep_TestMTDPosition.cc

```diff
@@ -212,27 +212,18 @@ void DD4hep_TestMTDPosition::analyze(const edm::Event& iEvent, const edm::EventS
         dump << "Box dimensions: " << convertCmToMm(mySens.halfX()) << " " << convertCmToMm(mySens.halfY()) << " "
              << convertCmToMm(mySens.halfZ()) << "\n";
 
-        char buf[256];
         DD3Vector x, y, z;
         fv.rotation().GetComponents(x, y, z);
-        size_t s = snprintf(buf,
-                            256,
-                            ",%12.4f,%12.4f,%12.4f,%12.4f,%12.4f,%12.4f,%12.4f,%12.4f,%12.4f,%12.4f,%12.4f,%12.4f",
-                            convertCmToMm(fv.translation().x()),
-                            convertCmToMm(fv.translation().y()),
-                            convertCmToMm(fv.translation().z()),
-                            x.X(),
-                            y.X(),
-                            z.X(),
-                            x.Y(),
-                            y.Y(),
-                            z.Y(),
-                            x.Z(),
-                            y.Z(),
-                            z.Z());
-        assert(s < 256);
-        dump << "Translation vector and Rotation components: " << buf;
-        dump << "\n";
+        dump << "Translation vector components: " << std::setw(14) << std::fixed << convertCmToMm(fv.translation().x())
```

Comment:
I think the format options `std::setw(14)` and `std::fixed` do not need to be reapplied, but will persist for the whole stream after being set the first time.

Reply:
@kpedro88 it does not seem fully the case, as I also would understand from https://en.cppreference.com/w/cpp/io/manip/setw 

```
        dump << "Translation vector components: " << std::setw(14) << std::fixed << fv.translation().x() << " "
             << std::setw(14) << std::fixed << fv.translation().y() << " " << std::setw(14) << std::fixed
             << fv.translation().z() << " "
             << "\n";
        dump << "Translation vector components: " << std::setw(14) << std::fixed << fv.translation().x() << " "
             << std::setw(14) << fv.translation().y() << " "
             << std::setw(14) << fv.translation().z() << " "
             << "\n";
        dump << "Translation vector components: " << std::setw(14) << std::fixed << fv.translation().x() << " "
             << fv.translation().y() << " "
             << fv.translation().z() << " "
             << "\n";
```

gives

```
Translation vector components:    1173.377418      79.984110       1.631200 
Translation vector components:    1173.377418      79.984110       1.631200 
Translation vector components:    1173.377418 79.984110 1.631200 
```

Reply:
Yes, you're right, the `setw` is needed each time. Thanks for checking.